### DISCUSSION TITLE
stable docker image is too out of date ?

### DISCUSSION THREAD

**Discussion by FarisZR** [simple-login/app#748] [Self-hosting question]
Hi I have been self-hosting simple login for a while now, and I was searching on how to block a sender from an alias.

All the options shown here: https://simplelogin.io/docs/getting-started/block-sender/
do not exist in the v3.4.0 tag, which is the latest apart from the CI builds which are mostly testing versions.

Is it time to release more up-to-date tag ? Which implements the v2 tag for all updates, which was discussed in https://github.com/simple-login/app/discussions/466#discussion-3363363

If the version used in simple login production is available to the public, we can just use it to follow the latest changes.

Also, Debian in the docker image is severely out of date.
![image](https://user-images.githubusercontent.com/35614734/149660083-f6f8a00b-918b-426a-ac2b-f0d95c6c0d0a.png)

**Comment by nguyenkims** [comment]
We will try to release a new version soon as the latest Docker image doesn't have recent features indeed. We don't have a fixed ETA though as releasing a new version requires to run some manual tests to make sure it doesn't break existing installations.

**Reply by FarisZR** [reply]
Any updates?
I'm ready to test updates on my instance to help with development.

**Reply by nguyenkims** [reply]
We plan to release the new version in March or April, please stay tuned!

**Reply by FarisZR** [reply]
Any updates ?

**Reply by jeisonbeast** [reply]
> We plan to release the new version in March or April, please stay tuned!

*Any update?

**Reply by FarisZR** [reply]
I think SL is busy with integrating with proton ATM.
I think a solution to this would be to just deploy the commit version used by SL it self
https://app.simplelogin.io/git

it's not tested for self-hosting though, so make sure you have backups, or just rebuild the docker image with a more up-to-date OS.

**Reply by DNCD** [reply]
> I think SL is busy with integrating with proton ATM.
> 
> I think a solution to this would be to just deploy the commit version used by SL it self
> 
> https://app.simplelogin.io/git
> 
> 
> 
> it's not tested for self-hosting though, so make sure you have backups, or just rebuild the docker image with a more up-to-date OS.

I try it, I get a lot of `Symbol '...' is unreachable ` have you tried it?

**Reply by 0xspade** [reply]
They moved it to May/June so probably this month or next month. Let's hope. Maybe if someone donates, they will update it sooner.

**Comment by FarisZR** [comment]
Looks like its partially fixed with the merger of #969 
There are now CI/CD builds for docker

https://hub.docker.com/r/simplelogin/app-ci
I don't know how stable / usable they are,
@nguyenkims can you give more details about the new build process implemented by proton?
